src/index.js

```diff
@@ -0,0 +1,115 @@
+import checkLocalStorage from './checkLocalStorage';
+import tabFactory from './tab_container';
+import './styles.css';
+
+const projectIdCounter = 1;
+const ttodos = [
+  {
+    projectId: 1,
+    projectTitle: 'Test project',
+    projectDescription: 'Here is my test project description',
+    projectTask: [
+      {
+        taskId: 'a1',
+        taskTitle: 'First Project',
+        taskDescription: 'First Project description ',
+        TaskDueDate: '1/1/1',
+        TaskPriority: 'high',
+      },
+      {
+        taskId: 'a2',
+        taskTitle: 'Test task2 for Test project',
+        taskDescription: 'Test task description for test project',
+        TaskDueDate: '1/2/3',
+        TaskPriority: 'medium',
+      },
+    ],
+  },
+];
+
+localStorage.setItem('todos', JSON.stringify(ttodos));
+
+const toddos = JSON.parse(localStorage.getItem('todos') || '[]');
+// console.log(toddos)
```

Comment:
## Delete commented out code

- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))